### DISCUSSION TITLE
Buffer is not defined

### DISCUSSION THREAD

**Discussion by camilo0119** [suren-atoyan/react-pwa#40] [Q&A] [answered]
`const base64 = {
  decode: (value: string) =>
    Buffer.from(value ?? "", "base64").toString("ascii"),
  encode: (value: string) =>
    Buffer.from(value ?? "", "ascii").toString("base64"),
};`

Uncaught ReferenceError: Buffer is not defined

<img width="318" alt="image" src="https://user-images.githubusercontent.com/16907221/198816031-664e3d49-9d81-4439-b296-6b42086102f3.png">

**Comment by suren-atoyan** [comment, marked as answer]
There is no `Buffer` in the EcmaScript specification.

We have `ArrayBuffer`, `SourceBuffer`, `AudioBuffer`, `WebGLBuffer`, but not `Buffer`.
